### DISCUSSION TITLE
nova.laravel.com down = UNREGISTERED in production

### DISCUSSION THREAD

**Discussion by sirikkoster** [laravel/nova-issues#4461] [General]
- Laravel Version: 9.17.0
- Nova Version: 4.9.2
- PHP Version: 8.1.2

### Description:

nova.laravel.com
-----
Error 522  
Ray ID: 71ead3657a86b71e • 2022-06-21 06:53:42 UTC
Connection timed out

laravel.com
-----
500 Internal Server Error

**Comment by baskinvolkan** [comment]
same here

**Comment by ziming** [comment]
cloudflare is down. that is why

**Comment by RealityGamin** [comment]
cloudflare is down https://www.cloudflarestatus.com/

![image](https://user-images.githubusercontent.com/100086757/174739602-2b827b6a-9e34-4423-9c37-771e7af77c50.png)

Its frustrating, I had to re-route all my servers traffic to bypass cloudflare to a different region :'(

**Comment by sirikkoster** [comment]
I also think that realtime license checking is not desirable in a production environment

**Reply by vesper8** [reply]
You're so right, which is why the first thing I do is null this out even though I'm a paying customer. It's one line of code.

**Comment by cweiske** [comment]
Yay for online license checks!

> GuzzleHttp\Exception\RequestException: cURL error 56: OpenSSL SSL_read: error:0A000126:SSL routines::unexpected eof while reading, errno 0 (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://nova.laravel.com/api/license-check
